src/components/App.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+  <div id="app">
```

Comment:
I believe this HTML will replace the div of the same name in index.html in dist, once the scripts are called in from main.js